mcs/class/monodoc/Monodoc.Ecma/EcmaDesc.cs

```diff
@@ -263,9 +255,18 @@ void ConstructCRef (StringBuilder sb)
 			if (DescKind == Kind.Type)
 				return;
 
+			if (ExplicitImplMember != null) {
+				sb.Append ('$');
```

Comment:
I should have suggested this earlier, but is there a reasonable way to use `#` everywhere _except_ in URLs, and in URLs we replace `#` with `$` or some other useful ASCII character (`,`? `!`?). That would at least allow us to be consistent with established practice.